test/conjugates_normal.jl

```diff
@@ -1,158 +1,197 @@
-# Conjugates for normal distribution
+# 
```

Comment:
?

Reply:
moved it to teh testset header (could keep the overall title but I think it's pretty clear)